packages/js/components/src/collapsible-content/style.scss

```diff
@@ -0,0 +1,23 @@
+.woocommerce-collapsible-content {
+
+	.woocommerce-collapsible-content__toggle {
```

Comment:
Wrapped the buttons in selectors so we don't accidentally modify button or svg styles in the content area.